README.md

```diff
@@ -1,2 +1,5 @@
 # hello-world
 Going through the tutorial to attempt to get competent on GitHub
+
+I just learned what a "branch" is (at last!).  I wish I'd know that earlier, it would have gotten me working on learning the GitHub setup sooner!
```

Comment:
I'm not sure the tutorial is doing a very accurate job of walking me through "pull requests"...my commit did not automatically create one though I did stumble across a compare screen without a pull request existing.  I may have just misunderstood some of the instructions though